assets/css/sensei-course-theme/content.scss

```diff
@@ -0,0 +1,23 @@
+.sensei-course-theme {
+	&__main-content {
+		padding: var(--content-padding);
+		margin-left: var(--sidebar-width) !important;
+
+		> * {
+			max-width: 900px;
+			margin-left: auto;
+			margin-right: auto;
+		}
+
+		.alignfull {
+			margin-left: -(--content-padding);
+			margin-right: -(--content-padding);
```

Comment:
Missed this in the previous PR: this is not valid syntax. For variable usage, it needs to be `var(--content-padding)`. For math, it also needs `calc( -1 * var( --content-padding ) )`. (Since this is just CSS, not a Sass compiler thing.)

Reply:
🤦 Thank you @yscik ! I didn't even realize I did that. Fixed 👍